spock-core/src/main/java/org/spockframework/compiler/WhereBlockRewriter.java

```diff
@@ -16,21 +16,27 @@
 
 package org.spockframework.compiler;
 
-import java.util.*;
```

Comment:
Unfortunately `.editorconfig` is too limited to really cover the whole spock formatting and not all code really has the correct one, however if you are using IntelliJ you can configure the formatting to only apply to your changed code.